File: core/src/test/resources/BreakpointTest.java
class BreakpointTest {
  public static void main(String[] args) {
    System.out.println("Breakpoint in main method");
    BreakpointTest test = new BreakpointTest();
    test.greet();
    System.out.println("Finished all breakpoints");
  }

  public BreakpointTest() {
    System.out.println("Breakpoin in constructor");
  }

  public void greet() {
    System.out.println("Breakpoint in method");
  }
}

File: core/src/test/resources/scala/BreakpointTest.scala
package scaladebug.test

object BreakpointTest {
  def main(args: Array[String]): Unit = {
    println("Breakpoint in main method")
    val h = new Hello
    h.greet()
    Hello // Force initialization of constructor
    println("Finished all breakpoints")
  }
  class Hello {
    def greet(): Unit = {
      println("Breakpoint in hello class")
      class InnerHello { println("Breakpoint in hello inner class") }
      new InnerHello()
      ()
    }
  }
  object Hello {
    println("Breakpoint in hello object")
    val a = 1
  }
}
